src/plugins/console/server/lib/spec_definitions/json/generated/cluster.put_component_template.json

```diff
@@ -0,0 +1,17 @@
+{
+  "indices.put_template": {
+    "url_params": {
```

Comment:
Per the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-component-template.html), the accepted query params are `create` and `master_timeout`.

Reply:
I went ahead and updated this.